### PR TITLE
3.9.0

### PR DESCRIPTION
**Pull request by patrickkwang**

### CODE REVIEW

File: strider/trapi.py
Comment (alongreyber) [inline strider/trapi.py:93]: Is there a reason we don't just return `kgraphs[0]`?

File: strider/trapi.py
Comment (alongreyber) [inline strider/trapi.py:97]: I don't really understand why this function is limited to merging two kgraphs, or why it returns early. Can you enlighten me?

File: strider/trapi_throttle/throttle.py
Comment (alongreyber) [inline strider/trapi_throttle/throttle.py:121]:
```suggestion
            self.logger.debug(
```
I think we should try to stick to local loggers and reduce use of `LOGGER`

File: strider/trapi_throttle/throttle.py
Comment (alongreyber) [inline strider/trapi_throttle/throttle.py:349]:
```suggestion
            self.logger.debug(f"Task cancelled: {task}")
```

**Review by alongreyber — CHANGES_REQUESTED** [review]
Haven't quite finished my review yet, but I will continue working on it later today. For now, here are some things that stood out to me.

File: strider/trapi.py
Comment (patrickkwang) [reply strider/trapi.py:94]: I left in statements after the return in order to hint to future developers that this was really meant as a temporary change until we find a less lossy solution to the merging speed problem. We merge only two kgraphs at a time because, in my limited experiments, this was the speediest solution. dict unpacking cannot be used in dict comprehension, or I'd have made it more general.

File: strider/trapi_throttle/trapi.py
Comment (patrickkwang) [reply strider/trapi_throttle/trapi.py:36]: I agree. In order to get this release out, I'd like to hold off on these cleanup steps for now.

File: strider/compatibility.py
Comment (patrickkwang) [reply strider/compatibility.py:14]: 6f79a3e476302ca2494bd49ce7755381320ed84a

File: strider/compatibility.py
Comment (patrickkwang) [reply strider/compatibility.py:55]: 6f79a3e476302ca2494bd49ce7755381320ed84a

File: strider/trapi.py
Comment (patrickkwang) [reply strider/trapi.py]: 1266bb9c0953890e0c31f0c4843649faab7bf513

File: strider/trapi_throttle/throttle.py
Comment (patrickkwang) [reply strider/trapi_throttle/throttle.py]: 7ea00f7b3b4ae5f233179d37b44458401f603945

File: strider/trapi_throttle/throttle.py
Comment (patrickkwang) [reply strider/trapi_throttle/throttle.py]: 7ea00f7b3b4ae5f233179d37b44458401f603945

**Review by alongreyber — APPROVED** [review]
Looks good. Can you open an issue for the nested directory structure after merging this?

**Pull request merged**
